shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/dml/delete.xml

```diff
@@ -104,54 +104,86 @@
         </where>
     </delete>
 
-    <delete sql-case-id="delete_with_special_comments_return_without_sharding_value">
-        <table name="t_order" start-index="33" stop-index="39" />
-        <where start-index="41" stop-index="54">
-            <expr>
-                <binary-operation-expression start-index="47" stop-index="54">
-                    <left>
-                        <column name="status" start-index="47" stop-index="52" />
-                    </left>
-                    <operator>=</operator>
-                    <right>
-                        <literal-expression value="1" start-index="54" stop-index="54" />
-                    </right>
-                </binary-operation-expression>
-            </expr>
-        </where>
-    </delete>
+<!--    <delete sql-case-id="delete_with_special_comments_return_without_sharding_value">-->
```

Comment:
We'd better delete these commented lines.

Reply:
Okay @wgy8283335, I'll update this and let you know.